tests/checkpoint/test_simple_checkpoint.py

```diff
@@ -209,7 +209,7 @@ def test_simple_checkpoint_raises_error_on_missing_slack_webhook_when_notify_on_
 def test_simple_checkpoint_raises_error_on_missing_slack_webhook_when_notify_on_is_not_default(
     empty_data_context, slack_notification_action, webhook
 ):
-    for condition in ["faliure", "success"]:
+    for condition in ["failure", "success"]:
```

Comment:
This actually looks like it impacts test outcomes. Great catch!

Reply:
Should I revert this change so the tests pass again?

Reply:
Hmm yes please! I'll add an issue/ticket around resolving this test and fix it up shortly.